lib/bamboo/phoenix.ex

```diff
@@ -51,6 +51,49 @@ defmodule Bamboo.Phoenix do
           |> render("text_email.text")
         end
       end
+
+  ## Full HTML Layout Example
+
+      # web/email.ex
+      defmodule Changelog.Email do
+        use Bamboo.Phoenix, view: Changelog.EmailView
+
+        def sign_in_email(person) do
+          base_email
+          |> to(person)
+          |> subject("Your Sign In Link")
+          |> assign(:person, person)
+          |> render(:sign_in)
+        end
+
+        defp base_email do
+          new_email
+          |> from("Rob Ot<robot@changelog.com>")
+          |> put_header("Reply-To", "editors@changelog.com")
+          |> put_html_layout({Changelog.LayoutView, "email.html"})
```

Comment:
Maybe add a code comment above this that says something like

```elixir
# This will use the "email.html.eex" file as a layout when rendering html emails. 
# Plain text emails will not use a layout unless you use `put_text_layout`
```